requirements.txt

```diff
@@ -1,13 +1,14 @@
 backports.functools_lru_cache==1.6.1; python_version == "2.7"
 dateparser==0.7.*
 pre-commit==1.20.*; python_version <= "3.5"
-pre-commit==2.1.*; python_version >= "3.6"
+pre-commit==2.3.*; python_version >= "3.6"
 pytest==4.6.*; python_version == "2.7"
-pytest==5.3.*; python_version >= "3.5"
+pytest==5.4.*; python_version >= "3.5"
 pytest-cov==2.8.*
-pytest-mock==2.0.*
+pytest-mock==2.0.*; python_version == "2.7"
+pytest-mock==3.1.*; python_version >= "3.5"
 python-dateutil==2.8.*
 pytz==2019.*
```

Comment:
`2020.1` is available if we want to change this to `2020.*`

Reply:
We can’t upgrade pytz until dateutil ships with the updated TZ database.